Calculator/calculator.py

```diff
@@ -1,22 +1,18 @@
-from numbers import Real
+from sum import sum
+from div import div
+import times
 
-def sum(num1, num2):
-    # Check if both inputs are numbers
-    if isinstance(num1, Real) and isinstance(num2, Real):
-        return num1 + num2
-    else:
-        print("This method only allows numeric values.")
+print("This will be our new calculator")
+print("Now we have submodules")
 
-def times(num1, num2):
-    # Check if both inputs are numbers
-    if isinstance(num1, Real) and isinstance(num2, Real):
-        return num1 * num2
-    else:
-        print("This method only allows numeric values.")
-                
-def subs(num1, num2):
-    # Check if both inputs are numbers
-    if isinstance(num1, Real) and isinstance(num2, Real):
-        return num1 - num2
-    else:
-        print("This method only allows numeric values.")
\ No newline at end of file
+print("-----------------------------------")
+print("Our first operation will be a sum")
+print("How much is 1 + 2 = " + sum(1,2))
```

Comment:
This file have this problem, can you resolve please?
Traceback (most recent call last):
 File "/Users/cardonita/Documents/Development/learning_python_with_me/Calculator/calculator.py", line 10, in <module>
    print("How much is 1 + 2 = " + sum(1,2))
          ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~
TypeError: can only concatenate str (not "int") to str